nifi-extension-bundles/nifi-aws-bundle/nifi-aws-kinesis/src/main/java/org/apache/nifi/processors/aws/kinesis/ConsumeKinesis.java

```diff
@@ -199,6 +202,17 @@ Ensure that the credentials provided have access to Kinesis, DynamoDB and (optio
             .identifiesControllerService(AwsCredentialsProviderService.class)
             .build();
 
+    static final PropertyDescriptor CONSUMER_TYPE = new PropertyDescriptor.Builder()
+            .name("Consumer Type")
+            .description("""
+                     The Consumer Type that is used to consume from the Kinesis records.
+                    (See processor's additional details for more information.)
```

Comment:
Recommend avoiding the mention of additional details, since it is generally implied that additional details contains more information.
```suggestion
                    Strategy for reading records from Amazon Kinesis streams.
```